packages/prerender/src/runPrerender.tsx

```diff
@@ -0,0 +1,93 @@
+import fs from 'fs'
+import path from 'path'
+
+import React from 'react'
+
+import babelRequireHook from '@babel/register'
+import prettier from 'prettier'
+import ReactDOMServer from 'react-dom/server'
+
+import { getPaths } from '@redwoodjs/internal'
+// @MARK: I had to do this to stop typescript from complaining
```

Comment:
@Tobbe worth looking into why router types don't load. It may just be lerna, and it'll fix it self in the next build.

Reply:
Fixed in https://github.com/redwoodjs/redwood/pull/1741/files